src/DiagramGenerator/Board.php

```diff
@@ -76,7 +77,11 @@ public function getImage()
      */
     protected function generateImage()
     {
-        $storage = new Storage($this->cacheDir, $this->pieceThemeUrl, $this->boardTextureUrl);
+        if ($this->config->hasThemeUrls()) {
+            $storage = new StorageNew($this->cacheDir);
+        } else {
+            $storage = new Storage($this->cacheDir, $this->pieceThemeUrl, $this->boardTextureUrl);
```

Comment:
What I usually prefer is to rename old class to `StorageLegacy`, because ultimately we would won't to delete the old class and new one should have the default (`Storage`) name. One less step during FF cleanup.